src/types.ts

```diff
@@ -78,7 +78,12 @@ export interface PublicMerchItem {
   uuid: Uuid;
   itemName: string;
   collection?: PublicMerchCollection;
-  picture: string;
+  merchPhotos: {
+    uuid: string;
+    uploadedPhoto: string;
+    uploadedAt: string;
+    position: number;
+  };
```

Comment:
make this an array:

```suggestion
  merchPhotos: {
    uuid: string;
    uploadedPhoto: string;
    uploadedAt: string;
    position: number;
  }[];
```

it type checked because the `noImplicitAny` TypeScript option was disabled, but it meant that `merchPhotos[0]?.uploadedPhoto` had type `any`